Diagnostic/DistroSpecific.py

```diff
@@ -246,7 +246,7 @@ def install_extra_packages(self, packages, with_timeout=False):
     def install_required_packages(self):
         # policycoreutils-python missing on Oracle Linux (still needed to manipulate SELinux policy).
         # tar is really missing on Oracle Linux 7!
-        return self.install_extra_packages(('policycoreutils-python', 'tar'), True)
+        return self.install_extra_packages(('policycoreutils-python-utils', 'tar'), True)
```

Comment:
would this exist in other distros too?

Reply:
The coverage seems to be fairly distributed but I believe the deciding factor is whether the system is using python2 or python3. 
https://pkgs.org/search/?q=policycoreutils-python-utils
https://pkgs.org/download/policycoreutils
because it is a direct replacement package in general, can we check for either package to satisfy the condition?